### DISCUSSION TITLE
I need the Immediate Props on Combobox

### DISCUSSION THREAD

**Discussion by alessiolilli** [tailwindlabs/headlessui#2833] [General]
I saw that [this](https://github.com/tailwindlabs/headlessui/pull/2686) Pull Request was merged, and i was asking myself when it will be realsed?

**Comment by tobeycodes** [comment]
@alessiolilli You can use insiders version

```
npm install @headlessui/react@insiders.
npm install @headlessui/vue@insiders.
```

@RobinMalfait Just curious. Is there a reason there hasn't been another release since Aug?

**Reply by alessiolilli** [reply]
I prefer not to use the insider version

**Reply by 0Chan-smc** [reply]
Why is that?

**Reply by alessiolilli** [reply]
Because this: "Note: The insiders build doesn't follow semver and therefore doesn't guarantee that the APIs will be the same once they are released."

**Comment by dansmithwwt** [comment]
+1 to this. Any updates on this being added to full release?
